backend/src/main/java/de/voelkldennis/backend/projects/ProjectService.java

```diff
@@ -27,5 +27,19 @@ public Optional<Project> getProjectWithId(String projectId) {
         return projectRepo.findByProjectId(projectId);
     }
 
-}
 
+    public Project deleteProjectWithId(String projectId) {
+        projectRepo.deleteByProjectId(projectId);
+        return null;
```

Comment:
Consider returning true on successful delete-operation (Makes testing easier) or make the method a void-method

Reply:
I changed the return type and the tests also.